packages/common/src/api/library.ts

```diff
@@ -9,66 +9,96 @@ import { reformatCollection } from 'store/cache/collections/utils/reformatCollec
 import { encodeHashId } from 'utils/hashIds'
 import { removeNullable } from 'utils/typeUtils'
 
-type GetLibraryAlbumsArgs = {
+type GetLibraryItemsArgs = {
   userId: number
   offset: number
   limit: number
   query?: string
   sortMethod?: full.GetUserLibraryAlbumsSortMethodEnum
   sortDirection?: full.GetUserLibraryAlbumsSortDirectionEnum
 }
+const COLLECTIONS_CACHE_OPTIONS = {
+  kind: Kind.COLLECTIONS,
+  schemaKey: 'collections',
+  idListArgKey: 'playlist_id'
```

Comment:
I don't think the `idListArgKey` does anything here. This is only useful when you're fetching a list of known playlist ids